README.md

```diff
@@ -503,6 +503,20 @@ import gym_minigrid
 
 You can train agents online using [colab notebook](https://colab.research.google.com/github/Stable-Baselines-Team/rl-colab-notebooks/blob/sb3/rl-baselines-zoo.ipynb).
 
+### Passing arguments in an interactive environment
+
+Modifying `sys.argv` is a possible way for passing arguments in an interactive computational environment (e.g: Jupyter Notebooks, IPython) without the need to launch a new Python process.
```

Comment:
Answering to https://github.com/DLR-RM/rl-baselines3-zoo/issues/297#issuecomment-1276313591

Ok, then I would add some kind of disclaimer:

"The zoo is not meant to be used from an interactive session. But ..."